### PR TITLE
Updates for the 2.2 release of interceptors

### PR DESCRIPTION
**Pull request by starksm64**
Draft of the Interceptors 2.2 spec update. Needs to wait on the CDI TCK for finalization.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of jakarta-wombat-spec-x.y.pdf
- [x] Specification HTML in the form of jakarta-wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [ ] For a Progress Review, that sufficient progress has been made on a Compatible Implementation and TCK, to ensure that the spec is implementable and testable.
- [x] For a [Release Review](https://www.eclipse.org/projects/handbook/#release-review), a summary that a Compatible Implementation is complete, passes the TCK, and that the TCK includes sufficient coverage of the specification. The TCK users guide MUST include the instructions to run the compatible implementations used to validate the release.
Instructions MAY be by reference.
   - [x] Updated release record
   - [x] Generated IP Log
   - [x] Email to PMC
   - [x] Start release review by emailing EMO 
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      https://jakarta.oss.sonatype.org/content/groups/staging/jakarta/interceptor/jakarta.interceptor-api/2.2.0/
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      [CDI 4.1 TCK](https://download.eclipse.org/ee4j/cdi/4.1/)
- [x] The URL of the compatibility certification request issue:
      [CDI 4.1 CCR](https://github.com/jakartaee/cdi/issues/789)
- [x] Specification JavaDoc in the wombat/x.y/apidocs directory. 
If desired, an optional second PR can be created to contain just the JavaDoc in the `apidocs` directory.

Note: If any item does not apply, check it and mark N/A below it.

### CODE REVIEW

**Comment by ivargrimstad** [comment]
Are the Interceptor signature tests run by Weld for CDI enough for a CCR for Interceptors?

https://raw.githubusercontent.com/wiki/jakartaredhat/cdi-tck/Jakarta-CDI-4.1-TCK-Weld-Results-17.md
https://raw.githubusercontent.com/wiki/jakartaredhat/cdi-tck/Jakarta-CDI-4.1-TCK-Weld-Results-21.md

**Comment by Emily-Jiang** [comment]
@starksm64 I have the following 2 comments:
1. change log is missing for this release. 
2. Compatible implementation says `tbd`. I think it should be weld-xxx.

**Comment by starksm64** [comment]
@Emily-Jiang The change log and CI have been updated.

**Comment by Emily-Jiang** [comment]
@starksm64 I don't see the 2.2 change log in the [spec doc](https://deploy-preview-702--jakartaee-specifications.netlify.app/specifications/interceptors/2.2/jakarta-interceptors-spec-2.2.html#changes-for-2-1)

**Comment by starksm64** [comment]
@Emily-Jiang the spec change log has been updated

**Comment by kazumura** [comment]
The link format of "The URL of the OSSRH staging repository" is incorrect.  
Maybe brackets ('[' and ']') are not needed.

File: interceptors/2.2/_index.md [hunk 1]
```diff
@@ -10,7 +10,12 @@ and other managed classes.
 ### New features, enhancements or additions
 <!-- List here -->
 * Updated dependencies for Jakarta EE 11
-* Add [standard accessor to interceptor bindings](https://github.com/jakartaee/interceptors/issues/100)
+  * Jakart Annotations to 3.0.0
```
Comment (Pandrex247) [inline interceptors/2.2/_index.md:13]: Typo
```suggestion
  * Jakarta Annotations to 3.0.0
```

**Comment by Emily-Jiang** [comment]
@starksm64 can you the above comments from @kazumura and @Pandrex247 before you merge this PR

**Comment by starksm64** [comment]
@Emily-Jiang this has been updated as per the comments and vote comments

**Comment by Emily-Jiang** [comment]
* On ballot completion, the specification committee mentor:
 - [ ] adds this final checklist to the main PR.
 - [ ] adds the `approved` label to the PRs, and sends out the Ballot Summary per this [template](#ballot-summary-email-template) to the [public Jakarta EE Specification Committee email list](jakarta.ee-spec@eclipse.org)
 - [ ] calculates the staged EFTL TCK signature and promotes it to the committee download area
  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job. Manually editing the jenkins Build Information will help identify the build (ie. Mail 2.0 or CDI 3.0).
 - [ ] merges the specification (and apidocs) PRs, ensuring the "date:" field in the _index.md file has an appropriate value to allow publishing.
 - [ ] updates the specification page with the ballot results. This is normally done via a separate PR that should be reviewed, approved, and merged.
 - [ ] notifies the EMO of the ballot results by email to [emo@eclipse-foundation.org](emo@eclipse-foundation.org). Just forward the ballot summary note sent earlier to the public Spec Committee email list.
 - [ ] creates an issue in the specification project that includes the following checklist for the specification project team:

**Pull request merged**